docs/gammapy.rst

```diff
@@ -0,0 +1,11 @@
+.. _gammapy:
+
+Gammapy Interoperability
+========================
+
+
```

Comment:
not sure if it's really that important but I would add at least one small paragraph explaining what we support in terms of gammapy classes

for example we support only PSF3D for now out of the 4 available from their API

Reply:
Have you actually looked at the generated docs? The three functions are immediately below this line and mention what classes are supported. It would just be repeating manually the autogenerated information.

Reply:
I cannot see the generated docs because they are crashing.

Reply:
Builds fine for me.